app/models/config_group_class.rb

```diff
@@ -10,6 +10,6 @@ class ConfigGroupClass < ActiveRecord::Base
   include AccessibleAttributes
 
   validates :puppetclass_id, :presence => true
-  validates :config_group_id, :presence => true,
-                              :uniqueness => {:scope => :puppetclass_id}
+  #validates :config_group_id, :presence => true,
```

Comment:
These two fields can be changed to `validate :puppetclass` and `validate :config_group` and tests will pass.